compiler/src/dmd/cond.d

```diff
@@ -691,6 +691,7 @@ extern (C++) final class VersionCondition : DVCondition
             case "LDC":
             case "linux":
             case "LittleEndian":
+            case "LoongArch64":
```

Comment:
this is missing a test for it not being a user (command line) settable version

Reply:
Done. Tests added.